package.json

```diff
@@ -92,6 +92,7 @@
     "react-native-shadow-2": "7.1.0",
     "react-native-share": "10.2.1",
     "react-native-svg": "15.4.0",
+    "react-native-url-polyfill": "2.0.0",
```

Comment:
Looks like is ~30kb which should be ok. but I think we should then start using it in other places where we try and build urls ? or we covered everywhere in this PR?

Reply:
This PR only covers that particular usecase we need for this feature.

The other place where we may want to consider using it, is in the rest client, to build query strings. We have already our own implementation for that, at it may be easier to use, so 0/5 (we could also just change the implementation of `buildQueryString` to use this library)

Another place is how we build profile picture urls and such, but again, probably it is ok with the current implementation.